packages/pangraph/src/pangraph/pangraph_node.rs

```diff
@@ -41,12 +41,12 @@ impl PangraphNode {
     }
   }
 
-  pub fn len(&self) -> usize {
-    self.position.1.saturating_sub(self.position.1)
-  }
-
-  pub fn is_empty(&self) -> bool {
```

Comment:
Therefore also this function was wrong. I thought about defining `is_emtpy` as checking if the start position coincides with the end position, but this is also wrong in one edge case: when the path is circular and composed of a single node, then the start/end position of the node can be the same even if the node is not empty.